signer/src/storage.rs

```diff
@@ -34,3 +35,30 @@ pub trait Write {
         block: &model::BitcoinBlock,
     ) -> impl Future<Output = Result<(), Self::Error>> + Send;
 }
+
+impl<'a, T> Read for &'a mut T
```

Comment:
Yeah, it's my habit of going with simple names and keeping namespace information when referring to them (e.g. `storage::Read` and `storage::Write`) - but that all breaks down since you need to bring traits into scope when using them (although you can do `use storage::Read as _` to mitigate this). I'll rename these to `DbRead` and `DbWrite` (since they will support more data than just bitcoin related data).

Reply:
Done ✅